docker/importer/importer.py

```diff
@@ -174,10 +174,6 @@ def import_new_oss_fuzz_entries(self, repo, oss_fuzz_source):
 
   def schedule_regular_updates(self, repo, source_repo):
     """Schedule regular updates."""
-    if (source_repo.last_update_date and
```

Comment:
This was the only place this property was being used. Let's just get rid of it completely (from models.py too).